src/Google.Cloud.Functions.Invoker/EntryPoint.cs

```diff
@@ -60,19 +61,77 @@ public static async Task<int> StartAsync(Assembly functionAssembly, string[] arg
             // doesn't matter much. Potentially catch exceptions during configuration, but let any
             // during web server execution propagate.
             var environment = FunctionEnvironment.Create(functionAssembly, args, ConfigurationVariableProvider.System);
-            IHostBuilder builder = Host.CreateDefaultBuilder(args)
+            await CreateHostBuilder(environment).Build().RunAsync();
+            return 0;
+        }
+
+        /// <summary>
+        /// Creates an <see cref="IWebHostBuilder"/> for the function type specified by
+        /// <typeparamref name="TFunction"/>. This is a convenience method for calling
```

Comment:
nit: I read "This is a covenience method for calling..." as if I had to something else after calling this method. Consider something like "This is a covenience method the same as calling..." Here and in the next one.